test/integration/user_can_create_account_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+
+class UserCanCreateAccountTest < ActionDispatch::IntegrationTest
+  test "user can create account and sees profile" do
+    items = create_list(:item, 2)
+
+    visit items_path
+
+    first(:button, "Add to Cart").click
+    assert page.has_content?("You added #{items.first.title} to your cart.")
+
+    visit item_path(items.last)
+
+    click_button "Add to Cart"
+    assert page.has_content?("You added #{items.last.title} to your cart.")
+
+    find("#shopping_cart").click
```

Comment:
Also, for this test, I think we only really need one item in the cart prior to visiting the cart. Adding items from multiple paths is covered in our other tests.